src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="https://www.moyo.ua/ua/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MoyoLogo"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple_1/"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/brands/samsung/"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/telecommunication/smart/"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/comp-and-periphery/"
+              class="nav_link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/gadgets/"
+              class="nav_link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/tablet_el_knigi/tablet/"
+              class="nav_link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/foto_video/photo_video/"
+              class="nav_link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="video">
+            <a
+              href="https://www.moyo.ua/ua/foto_video/video_tech/"
+              class="nav_link"
+            >
+              Video
+            </a>
+          </li>
```

Comment:
Remember to add empty lines between multiline sibling blocks of HTML for readability. However, there should not be a blank line between the `<ul>` and its first child `<li>`, nor between subsequent `<li>` elements.